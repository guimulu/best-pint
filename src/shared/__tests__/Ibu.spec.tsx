import * as React from "react";
import { createSnapshot } from "jest-react-fela";

import { Ibu } from "..";

describe("Ibu component green", () => {
  it("renders correctly", () => {
    const snapshot = createSnapshot(<Ibu ibu={10} />);
    expect(snapshot).toMatchSnapshot();
  });
});

describe("Ibu component yellow", () => {
  it("renders correctly", () => {
    const snapshot = createSnapshot(<Ibu ibu={20} />);
    expect(snapshot).toMatchSnapshot();
  });
});

describe("Ibu component red", () => {
  it("renders correctly", () => {
    const snapshot = createSnapshot(<Ibu ibu={30} />);
    expect(snapshot).toMatchSnapshot();
  });
});
