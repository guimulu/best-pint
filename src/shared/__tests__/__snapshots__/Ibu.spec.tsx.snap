// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Ibu component green renders correctly 1`] = `
".a {
  background-color: #21ab64;
}
.b {
  padding: 8px;
}
.c {
  position: absolute;
}
.d {
  left: 0;
}
.e {
  top: 36px;
}
.f {
  border-top-left-radius: 8px;
}
.g {
  border-bottom-right-radius: 8px;
}
.h {
  color: #fff;
}
.i::before {
  content: \\"IBU: 10\\";
}
.j::before {
  font-size: 16px;
}
.k::before {
  font-weight: bold;
}


<div className=a b c d e f g h i j k />;
"
`;

exports[`Ibu component red renders correctly 1`] = `
".a {
  background-color: #d3603a;
}
.b {
  padding: 8px;
}
.c {
  position: absolute;
}
.d {
  left: 0;
}
.e {
  top: 36px;
}
.f {
  border-top-left-radius: 8px;
}
.g {
  border-bottom-right-radius: 8px;
}
.h {
  color: #fff;
}
.i::before {
  content: \\"IBU: 30\\";
}
.j::before {
  font-size: 16px;
}
.k::before {
  font-weight: bold;
}


<div className=a b c d e f g h i j k />;
"
`;

exports[`Ibu component yellow renders correctly 1`] = `
".a {
  background-color: #e6cf01;
}
.b {
  padding: 8px;
}
.c {
  position: absolute;
}
.d {
  left: 0;
}
.e {
  top: 36px;
}
.f {
  border-top-left-radius: 8px;
}
.g {
  border-bottom-right-radius: 8px;
}
.h {
  color: #fff;
}
.i::before {
  content: \\"IBU: 20\\";
}
.j::before {
  font-size: 16px;
}
.k::before {
  font-weight: bold;
}


<div className=a b c d e f g h i j k />;
"
`;
